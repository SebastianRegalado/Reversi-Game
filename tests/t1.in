new 8
play 1 1
play 2 2
play 5 5
play 3 7
play 6 1
play 5 7
play 7 0
play 0 5
play 0 3
play 4 3
play 4 4
play 4 5
play 5 6
