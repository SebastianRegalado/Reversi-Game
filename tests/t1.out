--------
--------
--------
---BW---
---WB---
--------
--------
--------
--------
-B------
--------
---BW---
---WB---
--------
--------
--------
--------
-B------
--W-----
---BW---
---WB---
--------
--------
--------
--------
-B------
--W-----
---BW---
---WB---
-----B--
--------
--------
--------
-B------
--W-----
---BW--W
---WB---
-----B--
--------
--------
--------
-B------
--W-----
---BW--W
---WB---
-----B--
-B------
--------
--------
-B------
--W-----
---BW--W
---WB---
-----B-W
-B------
--------
--------
-B------
--W-----
---BW--W
---WB---
-----B-W
-B------
B-------
-----W--
-B------
--W-----
---BW--W
---WB---
-----B-W
-B------
B-------
---B-W--
-B------
--W-----
---BW--W
---WB---
-----B-W
-B------
B-------
---B-W--
-B------
--W-----
---BW--W
---WWW--
-----B-W
-B------
B-------
---B-W--
-B------
--W-----
---BW--W
---WWW--
-----BBW
-B------
B-------
